support-frontend/assets/components/socialShare/socialShare.jsx

```diff
@@ -24,29 +24,30 @@ type SocialMedia = {
   a11yHint: string,
 };
 
+type WindowFeatures = '' | 'menubar=no, toolbar=no, resizable=yes, scrollbars=yes, width=500, height=400';
 
 // ----- Setup ----- //
 
 const socialMedia: {
   [Platform]: SocialMedia,
 } = {
   facebook: {
-    link: 'https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fsupport.theguardian.com%2F%3FINTCMP%3Dsocial&t=',
+    link: 'https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fsupport.theguardian.com%2Fuk%2Fcontribute',
```

Comment:
these links are all hardcoded to the `uk` region. you should link to the geo-redirect URL https://support.theguardian.com/contribute

Reply:
Fixed